kafka/consumer.go

```diff
@@ -0,0 +1,46 @@
+package kafka
+
+import "github.com/Shopify/sarama"
+
+type Consumer interface {
+	Get() []byte
+}
+
+// Consumer Implementation
+type KafkaConsumer struct {
+	producer sarama.Consumer
+	topic    string
+}
+
+func NewConsumer(topic string, consumer sarama.Consumer) Consumer {
+	return &KafkaConsumer{
+		producer: consumer,
+		topic:    topic,
+	}
+}
+
+func (self *KafkaConsumer) Get() []byte {
+	/*consumer, err := NewConsumer([]string{"localhost:9092"}, nil)
```

Comment:
It's left over code. Originally I was going to make an interface for the consumer just as I did for the producer.